src/reducers/permanenceReducer.js

```diff
@@ -121,15 +121,18 @@ export default function permanence(state = initialState, action) {
         isUpdated: false
       };
     case 'UPDATE_FIELD':
-      let fields = state?.fields ?? [];
-      delete state?.errorsFormulaire?.errors?.filter(erreur => erreur?.[action.field.name])[0]?.[action.field.name];
-      delete fields?.filter(field => field.name === action.field.name)[0]?.value;
-      delete fields?.filter(field => field.name === action.field.name)[0]?.name;
-      fields?.push(action.field);
-      fields = nettoyageState(fields);
-      return {
-        ...state,
-        fields: fields
+      const updatedFields = state.fields.map(field =>
```

Comment:
j'arrive pas à voir ce qui pourrait faire le "is not a function" ) à cet endroit ?

Reply:
je n'ai pas réussi à reproduire l'erreur, j'ai cherché par elimination et il n'y a qu'ici que l'erreur pourrait potentiellement se déclencher